open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/authservlet/impl/util/Utils.java

```diff
@@ -250,4 +250,47 @@ private static List<Map<String, String>>  addAccList (JSONObject dataSet) {
         return accountData;
     }
 
+    /**
+     * Method to populate vrp data to be sent to consent page.
+     *
+     * @param request
+     * @param dataSet
+     * @return
+     */
+    public static Map<String, Object> populateVRPDataRetrieval(HttpServletRequest request, JSONObject dataSet) {
+
+        String selectedAccount = null;
+        Map<String, Object> returnMaps = new HashMap<>();
+
+        //Sets "data_requested" that contains the human-readable scope-requested information
+        JSONArray dataRequestedJsonArray = dataSet.getJSONArray(ConsentExtensionConstants.CONSENT_DATA);
```

Comment:
`dataRequestedJsonArray ` shall we update this variable name to match with `ConsentExtensionConstants.CONSENT_DATA`

Reply:
Also shall we update the comment in line 265?

Reply:
Fixed in [460b4c0]